core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -1038,6 +1059,13 @@ class SparkContext(config: SparkConf) extends Logging {
    * operation will create many references to the same object.
    * If you plan to directly cache, sort, or aggregate Hadoop writable objects, you should first
    * copy them using a `map` function.
+   * @param path directory to the input data files, the path can be comma separated paths
+   * as a list of inputs
+   * @param minPartitions suggested minimum number of partitions for the resulting RDD
+   * @param km implicitly found Class of the key associated with the `fm` implicit parameter
```

Comment:
Interesting, are these javadoc-able? a caller wouldn't really specify them explicitly. Does javadoc succeed when you build with this change? I don't know if these docs help much; they just say it's some implicit thing associated to another value. I wouldn't add docs just for completeness.